Monika After Story/game/pong.rpy

```diff
@@ -430,28 +579,502 @@ init -1 python in mas_pong:
-    m 1eua "At least we can still hang out with each other."
-    m 1hub "Ahaha!"
-    m 1eua "Thanks for letting me win, [player]."
-    m 1tku "Only elementary schoolers seriously lose at Pong, right?"
-    m 1hua "Ehehe~"
+# dialogtue shown right when monika loses
+label mas_pong_dlg_loser:
+    # adapts the dialog, depending on the difficulty, game length (bounces), games losses, wins and dialog response.
+    # the order of the dialog is crucial, the first matching condition is chosen, other possible dialog is cancelled in the process.
+    # todo: adapt dialog to affection
+    # todo: add randomized dialog
+    
+    $ monika_asks_to_go_easy = False
+    
+     
+    #Monika loses on purpose
+    if lose_on_purpose:
+        m 5hub "Ahahaha!"
+        m 1kua "Now we're even, [player]!"
+        $ lose_on_purpose = False
+        
+        
+    #Monika loses without hitting the ball
+    elif ball_paddle_bounces == 0:
+        if pong_last_responce_id == 1010: 
+            m 1rksdlb "Maybe I should try a bit harder..."
+        else:
+            m 1rksdlb "Ahaha, I guess I was a bit too slow there..."
+        
+        $ pong_last_responce_id = 1010
+              
+              
+    #Player starts playing seriously and wins after losing at least 3 times on purpose
+    elif instant_loss_streak_counter_before >= 3 and persistent.player_once_let_monika_win_on_purpose:
+        m 5tsu "So you are playing seriously now?"#
+        m 1huu "Let's find out how good you really are, [player]!"  
+        
+    
+    #Player wins after losing at least three times in a row
+    elif loss_streak_counter_before >= 3:
+        m 4eub "Congrats [player]!"#
+        m 2eua "I knew you would win a game after enough practise!"
+        m 4eua "Remember that skill comes mostly through repetitive training."
+        m 4hub "If you train long enough I'm sure you can reach everything you aim for!"
+        
+        
+    #Monika loses after saying she would win this time
+    elif powerup_value_this_game == DIFFICULTY_POWERUP:
+        m 6wuo "Wow..."
+        m 2tub "I was really trying this time, but you still won."
+        m 5hub "Way to go, [player]!"
+        
+        
+    #Monika loses after going easy on the player
+    elif powerup_value_this_game == DIFFICULTY_POWERDOWN:
+        m 5hub "Ehehehe!"
+        m 6esa "Good job, [player]!"
+        
+        
+    #Monika loses after going even easier on the player
+    elif powerup_value_this_game == DIFFICULTY_POWERDOWNBIG:
+        m 1hsb "I'm glad you won this time, [player]."
+        
+    
+    #Monika loses by a trickshot
+    elif pong_angle_last_shot >= 0.9 or pong_angle_last_shot <= -0.9:
+        if pong_last_responce_id == 1070:     
+            m 2dfd "[player]!" 
+            m 2tkc "This is kind of unfair..." 
+            m 2ekp "..."
+            m 1hksdlb "Sorry, [player]."
+            m 1lsc "Losing like this kind of gets to me."
```

Comment:
Would Monika actually say this?

Reply:
I have yet to see her in any comparable situation to make accurate assumptions.
This is a pretty rare event.
You can keep her one dimensional if you like, rephrase it into something positive/neutral then.